NetworkExplorationCode/FeatureVectorVisualizer.py

```diff
@@ -0,0 +1,95 @@
+import pandas as pd
+from sklearn.decomposition import PCA
+import plotly.express as px
+from sklearn.manifold import TSNE
+import plotly.figure_factory as ff
+import numpy as np
```

Comment:
FYI, order of imports is important for efficiency. for example, numpy is used by pandas, so importing numpy before pandas will speed up pandas loading. Not important on our scale, but just so you know.